app/Http/Controllers/API/AdminController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\AdminRequest;
+use App\Http\Resources\AdminResource;
+use App\Models\Admin;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AdminController extends Controller
+{
+    public function registerAdmin(AdminRequest $request)
```

Comment:
@Babita3331 The whole admin registration seems to be flawed. If the endpoint for admin registration is exposed in the documentation, a user is able to create multiple admin accounts. Plus, the user can use other people's emails without them knowing, to register and make changes.
If there will be just one admin, seed it directly to the database. If there will be multiple admins ACL should be implemented